src/labs_examples/conditions_loops/labs/Exercise_02.java

```diff
@@ -10,15 +10,42 @@
  *      Bonus Tricky Challenge: Use a "nested-if" statement.
  *
  */
-
+import  java.util.Scanner;
 public class Exercise_02 {
 
     public static void main(String[] args) {
 
+
         // 1) create scanner (don't forget to import Scanner!)
+        Scanner input = new Scanner(System.in);
         // 2) prompt user
+        System.out.println("Enter a number 1 : 8 ( Each number representing " +
+                        "a day of the week (1) being monday (7) being sunday. (8) other: ");
         // 3) assign input to variable as int
+        int weekNum = input.nextInt();
         // 4) write completed code here
 
+        if(weekNum == 1){
+            System.out.println("Today is monday");//just curios is there better way to do it with a for loop.
```

Comment:
Well, since the amount of options is limited and known exectly during coding, the best option is to use `switch` with predetermined caes.

If the options were not known or dynamic the best option is a `map`, whitch you will study in `25) Abstract Data Structures` as it is a bit more complicated :)